components/TakeAction/GiveSkills.js

```diff
@@ -0,0 +1,180 @@
+import ProjectGoalIcon from './ProjectGoalIcon';
+import BulletPointIcon from './BulletPointIcon';
+
+import styles from '../../constants/TakeAction/GiveSkill';
+
+const NeededSkills = props => {
+  var skillsBubbles = [];
+
+  props.neededSkills.forEach( skill => {
+    skillsBubbles.push(
+      <View style={styles.skill_bubble}>
+        <Text style={styles.skill_label}>{skill.skill}</Text>
+      </View>
+    )
+  });
+
+  return <View style={styles.skill_bubble_container}>{skillsBubbles}</View>
+}
+
+const Goal = props => {
+  return (
+    <View style={styles.goal_container}>
+      <BulletPointIcon />
+      <View style={styles.goal_description_container}>
+        <Text style={styles.goal_title}>{props.title}</Text>
+        <Text style={styles.goal_text}>{props.text}</Text>
+      </View>
+    </View>
+  );
+}
+
+const NeededSkillsDetailContainer = props => {
+  var neededSkillsDetails = [];
+
+  props.data.skilled_impact_requests.forEach( skill => {
+    neededSkillsDetails.push(<NeededSkillsCard skill={skill} navigation={props.navigation}/>)
+  })
+
+  return <View >{neededSkillsDetails}</View>
+}
+
+class NeededSkillsCard extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.skill = props.skill;
+
+    this.goals = [];
+    if (this.skill.project_goals) {
+      this.skill.project_goals.forEach(goal => {   
+        this.goals.push(<Goal title={goal.goal_title} text={goal.description} />);
+      })
+    }
+  }
+
+  state = {
+    expanded: false
+  }
+
+  handleExpand = () => {
+    this.setState({
+      expanded: !this.state.expanded 
+    })
+  }
+
+  render() {
+    return (
+      <View style={styles.needed_skills_card}>
+        <View style={styles.section_header}>
+          <Text style={styles.skill_needed_header}>Skill Needed:</Text>
+          <View>
+            <View style={styles.skill_bubble}>
+              <Text style={styles.skill_label}>{this.skill.skill}</Text>
+            </View>
+          </View>
+          <View style={styles.arrow_container}>
+            {this.state.expanded ? 
+              <TouchableOpacity onPress={this.handleExpand}>
+                <DownArrowIcon />
+              </TouchableOpacity>
+              
+              : <TouchableOpacity onPress={this.handleExpand}>
+                <RightArrowIcon />
+              </TouchableOpacity>
+            }
+          </View>
+        </View>
+
+        {this.state.expanded &&
+          <View>
+            {this.goals.length > 0 && 
+              <View style={styles.section_header}>
+                <ProjectGoalIcon />
+                <View style={styles.project_goal_title_container}>
+                  <Text style={styles.project_goal_title}>Project Goals</Text>
+                </View>  
+              </View>
+            }
+            {this.goals}
+            <TouchableOpacity
+              style={ styles.buttonTouch }
+              onPress={() => {
+                // TODO: Handle application process
```

Comment:
Can we show a quick toast or something that says "Coming Soon"? Don't want the UX to be confusing if we have to release before the application page is done.